michael-msrgnt/hw_2/5-random-address-generator-bonus/app.js

```diff
@@ -0,0 +1,12 @@
+var numby = Math.ceil(Math.random() * 50000);
+var street = ["Poppy Street", "Elm Avenue", "dPoop View"];
+var city = ["Funk City", "Poop Town", "Los Angeles"];
+var state = ["CA", "NY", "MI", "XX"];
```

Comment:
I am skeptical about the validity of these addresses, but the code runs as expected so I see no reason to look into the matter further. :woman_shrugging: 

Did you hear the joke about the zoo with only one exhibit, and in it was a dog? It was a shih tzu. Aha hahaha ha...